src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -97,9 +98,144 @@ def custom_REE_plant_currency_units():
 @declare_process_block_class("QGESSCosting")
 class QGESSCostingData(FlowsheetCostingBlockData):
     # Register currency and conversion rates based on CE Index
-    # register_idaes_currency_units()
+
     custom_REE_plant_currency_units()
 
+    # set CONFIG
+
+    CONFIG = FlowsheetCostingBlockData.CONFIG()
+
+    CONFIG.declare(
+        "discount_percentage",
+        ConfigValue(
+            default=None,
+            domain=float,
+            description="rate of return used to discount future cash flows "
+            "back to their present value. The NETL QGESS recommends setting "
+            "the discount rate as the calculated after-tax weighted average "
+            "cost of ccapital (ATWACC). For the UKy case study, the ATWACC is"
+            "5.77%.",
```

Comment:
For 1, Users should specify the discount percentage and project lifetime, as these two inputs have an outsized impact on the NPV calculation and are project-specific. I think it better to enforce that users set values for these two inputs and include the comment about the typical value rather than have defaults.

I'll update the doc strings to address 2 and 3. All percentage inputs are assumed to be percentage values (e.g. 5.77 means 5.77%).

Reply:
Regarding 1 then, I would remove the comment about the value for the UKy case study, as it is too-specific for a general tool.